odoo/modules/registry.py

```diff
@@ -367,6 +367,29 @@ def concat(seq1, seq2):
 
         return triggers
 
+    @lazy_property
+    def relational_trigger_fields(self):
+        '''
+        Return a set of fields that, when modified, cause relational fields to
+        change, and thereby indirect field recomputations.
```

Comment:
```suggestion
        Return the set of fields that, when modified, cause relational fields to
        change (directly or indirectly via recomputation).
```

Reply:
This one should be correct:
```suggestion
        Return the union of the set of relational fields that are dependencies
        of other fields, with the set of non-relational fields that are
        dependencies of relational fields.
```